.github/workflows/ci.yaml

```diff
@@ -47,3 +47,31 @@ jobs:
         if: success() || failure() # still run black if above checks fails
         run: |
           black --check --verbose .
+  regression:
+    runs-on: ubuntu-22.04
+    permissions:
+      id-token: write # This is required for requesting the JWT
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install .
+      - name: Configure AWS Credentials
+        uses: mcblair/configure-aws-profile-action@v0.1.1
```

Comment:
See https://github.com/smart-on-fhir/cumulus-library/issues/113 for a discussion of this marketplace action.